source/content/wordpress-best-practices.md

```diff
@@ -117,3 +117,29 @@ This method has the advantage of being toggleable without deploying code, by act
   ```
 
 1. Commit your work, deploy code changes then activate the plugin on Test and Live environments.
+
+## Security Headers
+
+Pantheon's Nginx configuration cannot be modified to add security headers, and most solutions (including plugins) written about security headers for WordPress involve modifying the `.htaccess` file for Apache-based platforms.
+
+Adding code like the example below in a plugin or to your theme can help add security headers for WordPress sites on Pantheon, or any other Nginx-based platform.
+
+These headers apply when accessing site pages or the REST API of WordPress, but not when directly accessing assets like `https://example.com/wp-content/uploads/2020/01/sample.json`.
+
+The code below is only an example to get you started. You'll need to modify it to match your needs, especially the Content Security Policy. Tools like [SecurityHeaders.com](https://securityheaders.com) can help to check your security headers and links to additional information on where to improve your security header profile.
+
+```php
+function additional_securityheaders( $headers ) {
+  if ( ! is_admin() ) {
+    $headers['Referrer-Policy']             = 'no-referrer-when-downgrade'; //This is the default value, the same as if it were not set.
```

Comment:
Maybe consistent spacing for the code sensitive user

Reply:
Addressed in https://github.com/pantheon-systems/documentation/pull/6159/commits/1c1beec0abb3bc2371a510f8b3d314d7f4432c0f

Reply:
👍